packages/api/src/controllers/clip.ts

```diff
@@ -160,4 +165,100 @@ app.post("/", validatePost("clip-payload"), async (req, res) => {
   });
 });
 
+const fieldsMap = {
+  id: `asset.ID`,
+  createdAt: { val: `asset.data->'createdAt'`, type: "int" },
+  updatedAt: { val: `asset.data->'status'->'updatedAt'`, type: "int" },
+  userId: `asset.data->>'userId'`,
+  playbackId: `asset.data->>'playbackId'`,
+  sourceType: `asset.data->'source'->>'type'`,
+  sourceSessionId: `asset.data->'source'->>'sessionId'`,
+  sourceAssetId: `asset.data->'source'->>'assetId'`,
+} as const;
+
+app.get("/:id", authorizer({}), async (req, res) => {
```

Comment:
As discussed offline, let's move this to the `/asset` root since this is a GET an API on the asset resource and there's no separate "clip" object. (the POST is fine for now, since the input can be other things than an asset so it could be confusing inside /asset as well)

This is ok to merge as is though, but should not be documented to end-users and we should move it before doing so